src/components/MovieListItem.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <div class='movie-item'>
+        <p><span class="enhanced-text">Title:</span> {{ title }}</p>
+        <p><span class="enhanced-text">Genre:</span> {{ genre }}</p> 
+        <p><span class="enhanced-text">Year:</span> {{ year }}</p>
+        <p><span class="enhanced-text">Description:</span><br>{{ description }}</p>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'MovieItem',
+    props: {
+        movie: Object,
+    },
+    data() {
+        return {
+            title: this.movie.title,
```

Comment:
Instead of having a data for this, you could use the movie prop in the template:
In example: instead of {{ title }}, you could have {{ movie.title }}, that way we can get rid off the whole data :) It will be reactive, so any time the movie changes, you'll have the most up to date value